rllib/policy/tests/test_policy_map.py

```diff
@@ -0,0 +1,143 @@
+import gym
+import numpy as np
+import time
+from typing import Optional
+import unittest
+
+import ray
+from ray.rllib.algorithms.ppo import PPOConfig, PPOTF2Policy
+from ray.rllib.policy.policy_map import PolicyMap
+from ray.rllib.utils.test_utils import check
+from ray.rllib.utils.tf_utils import get_tf_eager_cls_if_necessary
+
+TIME_NO_SWAPS: Optional[float] = None
+TIME_SWAPS: Optional[float] = None
+
+
+class TestPolicyMap(unittest.TestCase):
```

Comment:
New test case checking, whether swapping (`a = A.get_state()` -> `B.set_state(a)` -> B becomes A) vs no-swapping (re-creating of policies) is much faster.